lxd/db/images.go

```diff
@@ -556,6 +558,64 @@ func (c *Cluster) GetImageFromAnyProject(fingerprint string) (int, *api.Image, e
 	return object.ID, &image, nil
 }
 
+// getImagesByFingerprintPrefix returns the images with fingerprints matching the prefix. Optional filters 'project' and 'public' will be included if not nil.
```

Comment:
@masnax small nit, we tend to try and keep function comment width within that of the github viewport.